### DISCUSSION TITLE
Closing original source file before I'm finished

### DISCUSSION THREAD

**Discussion by sujfuy** [mifi/lossless-cut#2034] [General]
When I do a cut in Lossless, it auto closes my target file and I'm not finished.  Nothing I do seems to stop this and it's a lot of trouble to go back and find the exact location again where I left off.

**Comment by mifi** [comment]
you mean when you export? check settings -> "Cleanup files after export"

**Reply by sujfuy** [reply]
I don't see anything here that fixes the closing of the original file. In advanced, there's an option to close existing opened file which is checked but I cannot uncheck it.

**Comment by sujfuy** [comment]
Well I figured out how to select the start and stop of multiple clips in one file instead of using "i" and "0"

**Comment by sujfuy** [comment]
Well I posted this question again and didn't realize I had already ask this, but apparently never got an answer anyway.  I'm just really surprised that the program closes after exporting a clip.  I need to keep it open in the same place.

**Reply by mifi** [reply]
go to "settings" -> "Cleanup files after export"

**Reply by sujfuy** [reply]
Pretty sure I checked that but I will again.

**Reply by sujfuy** [reply]
Close existing file is checked but cannot uncheck.

**Reply by mifi** [reply]
"Close currently opened file" can **only** be unchecked when both "Trash auto-generated files" and "Trash original source file" are also unchecked. does that help?

**Reply by sujfuy** [reply]
I'll try it and see... I was trying to update to the newest version but can't seem to accomplish it.  Won't let me extract it to program folder, can only seem to use it from my download folder which I do not want it there.
Thanks

**Reply by sujfuy** [reply]
I've got everything unchecked and it still closes my project/video after cutting out a clip and exporting.

**Reply by sujfuy** [reply]
Nevermind, I finally got it. Still can't update the program though.  Thanks!
